docs/getting-started.md

```diff
@@ -9,17 +9,6 @@ In this guide we will show you how to create a tenant and connect the Beyond Ide
 ### Create a Tenant
 Navigate here to [Create a Tenant](https://www.beyondidentity.com/developers/signup).
 
-Once your tenant has been created with Beyond Identity, you can continue to create an API token. 
```

Comment:
Why is this deleted?

Reply:
I thought we didn't need this anymore? I can put it back